Makefile

```diff
@@ -3,7 +3,7 @@ MINIMAL_NOTEBOOK_IMAGE := quay.io/jupyter/minimal-notebook:python-3.11
 MINIMAL_CONTAINER_NAME := testminimal
 
 # Test container being developed.
-TEST_IMAGE_TAG := testmarble
+TEST_IMAGE_TAG := marble.v1
```

Comment:
note to do proper tagging (semantic versioning etc.)

Reply:
Changed

Reply:
Okay I see what you mean. I can revert the name back, I overlooked it. I meant it was resolved because I added the deployment related variables which generate the correct names and tags for deployed versions.